src/imp/libc/process/auxv.rs

```diff
@@ -52,3 +52,15 @@ pub(crate) fn linux_execfn() -> &'static ZStr {
         zstr!("")
     }
 }
+
+/// Initialize process-wide state.
+#[cfg(any(
+    target_vendor = "mustang",
+    not(any(target_env = "gnu", target_env = "musl"))
+))]
+#[inline]
+#[doc(hidden)]
+pub(crate) unsafe fn init(_envp: *mut *mut u8) {
+    // Nothing to do. This is the libc backend, and libc does the
```

Comment:
If it does nothing and libc does the initialization for us, why do you declare it?

Reply:
Just to keep the API consistent between the linux_raw backend, where this function does so something, and the libc backend here, which doesn't. This allows users to switch between the backends without changing their code.